### DISCUSSION TITLE
Need a search in my repositories

### DISCUSSION THREAD

**Discussion by g7i** [community/community#18368] [Mobile]
Need a search in my repositories. The search on home searches across all the GitHub repositories.

Thanks

**Comment by ale-lit** [comment]
Hi! Search on this page working only with your repositories: https://github.com/g7i?tab=repositories

![image](https://user-images.githubusercontent.com/91823697/173200031-6f818c3d-0560-4282-bbbb-c2bf7f566771.png)

**Reply by GuScarpim** [reply]
Easy

**Reply by g7i** [reply]
Sorry for the confusion, I meant for GitHub Android App.

**Comment by g7i** [comment]
There is no search button on the repositories screen

**Reply by g7i** [reply]
![IMG_20220612_074931.jpg](https://user-images.githubusercontent.com/46393634/173211908-6567bde1-5ad3-4749-95d2-63c757e3395b.jpg)

**Comment by melMass** [comment]
Is there a way to follow up on these feature requests?
I badly need repo/stars search on mobile too!
Cheers

**Reply by iskae** [reply]
Hi @melMass 👋 

Thanks a lot for taking your time and leaving this feedback. Also many thanks to others that have upvoted 🙇 

We already have an internal tracking issue for this 🎉 I have also included your feedback in that issue. Please stay tuned for our upcoming releases 😊 

Thanks!
